src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/DefaultRaptorTransfer.java

```diff
@@ -0,0 +1,20 @@
+package org.opentripplanner.routing.algorithm.raptoradapter.transit;
+
+import org.opentripplanner.transit.raptor.api.transit.RaptorTransfer;
+
+public record DefaultRaptorTransfer(
+  int stop,
+  int durationInSeconds,
+  int generalizedCost,
+  Transfer transfer
```

Comment:
We would probably get a bit better performance if we did not include the Transfer - instead we could make an  index or map. But this is an improvement anyway - very glad to see these classes go...